users/ridingqwerty/process_records.c

```diff
@@ -10,23 +13,477 @@ bool process_record_secrets(uint16_t keycode, keyrecord_t *record) {
   return true;
 }
 
-uint16_t user_key_timer;
+/*
+typedef union {
+    struct {
+        uint8_t aesthetic : 1,
+                spongebob : 1,
+                uppercase : 1,
+                australia : 1,
+                unusued   : 4;
+    };
+    uint8_t all;
+} mode_config_t;
+*/
+mode_config_t mode = {.all = 0U};
+
+bool randword_seed  = false;
+uint8_t user_mod_state;
+uint16_t rand_key;
+uint16_t temp_keycode;
+uint32_t user_key_timer;
+
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef CONSOLE_ENABLE
+  uprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+#endif
+
+  temp_keycode = keycode;
+  if ((mode.all != 0U) && ((keycode != DB(TAB) || keycode != ED(ESC)) && ((keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) || (keycode >= QK_LAYER_TAP && keycode <= QK_LAYER_TAP_MAX)))) {
+      temp_keycode &= 0xFF;
+  }
+
+  if (mode.spongebob) {
+    switch(temp_keycode) {
+      case KC_A ... KC_Z:
+        if (record->event.pressed) {
+          (mode.uppercase ^= 1) == 0 ? tap_code16(S(temp_keycode)) : tap_code(temp_keycode);
+        }
+        return false;
+    }
+  }
+
+  if (mode.aesthetic) {
+    switch(temp_keycode) {
+#if defined(UNICODE_ENABLE) || defined(UNICODEMAP_ENABLE)
+      case KC_A:
+        if (record->event.pressed) {
+          send_unicode_hex_string("039B");
+          tap_code(KC_SPC);
+        }
+        return false;
+      case KC_E:
+        if (record->event.pressed) {
+          send_unicode_hex_string("039E");
+          tap_code(KC_SPC);
+        }
+        return false;
+#else
+      case A:
+      case E:
+        if (record->event.pressed) {
+          register_code(KC_LSFT);
+          tap_code16(temp_keycode);
+          unregister_code(KC_LSFT);
```

Comment:
You could just do 
```suggestion
          tap_code16(LSFT(temp_keycode));
```

Reply:
I’ve tested this already and believe it or not, things get screwy.

Reply:
For giggles I amended it as you suggested and it works just fine now; I have no idea what had me addled previously.